Makefile

```diff
@@ -26,10 +28,12 @@ KUBECTL ?= $(CLUSTER_DIR)/kubectl.sh
 CLUSTER_UP ?= $(CLUSTER_DIR)/up.sh
 CLUSTER_DOWN ?= $(CLUSTER_DIR)/down.sh
 CLI ?= $(CLUSTER_DIR)/cli.sh
-SSH ?= $(CLI) ssh
+SSH ?= $(CLUSTER_DIR)/ssh.sh
```

Comment:
How is this change related?

Reply:
Little boy scout here, but I can change it back.